### DISCUSSION TITLE
Why doesn't the battery icon use the Win10 icon on Taskbar10 on Windows 11?

### DISCUSSION THREAD

**Discussion by Zycelz** [valinet/ExplorerPatcher#2136] [Q&A] [answered]
wtf happened to it, it doesn't look like the win10 one
![Screenshot 2023-05-18 034404](https://github.com/valinet/ExplorerPatcher/assets/105969023/ef51e0cd-5ddf-4fa4-a945-8f9790adc3a3)

**Comment by Poopooracoocoo** [comment]
Also see suggestion https://github.com/valinet/ExplorerPatcher/discussions/1407

**Comment by Jevil7452** [comment, marked as answer]
This is Windows 11's fault, as they actually updated the battery icon in batmeter.dll.
